packages/shared/src/components/cards/v1/ArticlePostCard.tsx

```diff
@@ -139,7 +139,7 @@ export const ArticlePostCard = forwardRef(function PostCard(
                 loading="lazy"
                 data-testid="postImage"
                 {...(isVideoType && {
-                  wrapperClassName: 'mt-4 mobileXL:w-40 mobileXXL:w-56',
+                  wrapperClassName: 'mt-4 mobileXL:w-40 mobileXXL:w-56 !h-fit',
```

Comment:
This is changing it only in ArticlePostCard for v1, did we want to change for Collections, Shared, Welcome cards as well in V1 - or those are not affected?

Reply:
Let me deploy in preview to further check.

Reply:
For Welcome, and Collections, we should not need the fix for v1 there since the type itself can never be video. For Shared, I have now pushed the same just to be on the safe side.